### ISSUE TITLE
The UILabel extension increases font size, but not enough on the iPad

### ISSUE THREAD

**Issue by Sweeper777** [tbaranes/FittableFontLabel#9]
I am using an `LTMorphingLabel` to display some text that I want to fill the screen with. So I call `fontSizetoFit()` on the `LTMorphingLabel`. And this is the result, viewed in the view hierarchy:

<img width="288" alt="screen shot 2016-07-28 at 8 30 40 am" src="https://cloud.githubusercontent.com/assets/13585047/17197296/2a19a1e0-549e-11e6-84e2-4b3e5a578b25.png">

The selected view is the frame of the `LTMorphingLabel`. As you can see, the text could have been much larger!

This only seems to be a problem on iPads though.

**Comment by tbaranes** [comment]
Do you know what is the final size of the label? By default, the max value will be 100, can you try to customise it by using `maxFontSize`?

``` swift
aLabel.fontSizeToFit(maxFontSize: 500)
```

**Comment by Sweeper777** [comment]
Thank you, it works. I was kind of confused because in the declaration of `fontSizeToFit`,

```
    public func fontSizeToFit(maxFontSize maxFontSize: CGFloat = CGFloat.NaN, minFontScale: CGFloat = 0.1,rectSize: CGSize? = nil)
```

The default value for `maxFontSize` is `NaN`! I thought that meant it can be infinitely large! Well, now I see this line:

```
let maxFontSize = maxFontSize.isNaN ? 100 : maxFontSize
```

It's pretty confusing in the README.md too. It says that the default is "the current font size". Can you just change that to 100?

**Comment by tbaranes** [comment]
You are right, I will update it! Thanks for the report.

**Comment by tbaranes** [comment]
Will be fixed by #10
